code/modules/power/apc.dm

```diff
@@ -107,12 +110,12 @@
 	var/failure_timer = 0
 	var/force_update = 0
 	var/updating_icon = 0
-	var/secret = FALSE // If true, it won't show up on the alert computer.
 	var/global/list/status_overlays_lock
 	var/global/list/status_overlays_charging
 	var/global/list/status_overlays_equipment
 	var/global/list/status_overlays_lighting
 	var/global/list/status_overlays_environ
+	var/hidden = 0 //If power alarms from this APC are visible on consoles

```

Comment:
Nitpick: I'd rather this be FALSE, if at all possible.

Reply:
Are we switching from a 0/1 system to a FALSE/TRUE system, finally?